neon_data_models/models/api/node_v1/__init__.py

```diff
@@ -0,0 +1,154 @@
+# NEON AI (TM) SOFTWARE, Software Development Kit & Application Development System
+# All trademark and other rights reserved by their respective owners
+# Copyright 2008-2024 Neongecko.com Inc.
+# BSD-3
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+# 1. Redistributions of source code must retain the above copyright notice,
+#    this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions and the following disclaimer in the documentation
+#    and/or other materials provided with the distribution.
+# 3. Neither the name of the copyright holder nor the names of its
+#    contributors may be used to endorse or promote products derived from this
+#    software without specific prior written permission.
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+# CONTRIBUTORS  BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA,
+# OR PROFITS;  OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+# LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE,  EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+from datetime import datetime, timedelta
+from pydantic import Field
+from typing import List, Literal, Optional, Annotated, Dict
+
+from neon_data_models.enum import UserData, AlertType, Weekdays
+from neon_data_models.models.base import BaseModel
+from neon_data_models.models.base.messagebus import BaseMessage, MessageContext
+
+
+class AudioInputData(BaseModel):
+    audio_data: str = Field(description="base64-encoded audio")
+    lang: str = Field(description="BCP-47 language code")
+
+
+class KlatResponse(BaseModel):
+    sentence: str = Field(description="Text response")
+    audio: Dict[Literal["male", "female"], Optional[str]] = Field(
+        description="Mapping of gender to b64-encoded audio")
+
+
+class AudioInputResponseData(BaseModel):
+    parser_data: dict
+    transcripts: List[str]
+    skills_recv: bool
```

Comment:
Note that these are used as annotations within this module and are intentionally excluded from `__all__` because they are not intended to be used directly